lib/apiservers/portlayer/swagger.json

```diff
@@ -1748,6 +1748,63 @@
 				}
 			}
 		},
+		"/containers/{handle}/name": {
+			"put": {
```

Comment:
Oh, I am sorry, I forgot that PUT can also mean update in terms of action. And UPDATE is actually PATCH.
PATCH seems a better fit here because of:
```
GET	Read
POST Create
PUT	Update/Replace
PATCH	Update/Modify
DELETE	Delete
```
We are really renaming, so it is really an update or modification not a replacement that PUT can mean.